src/pretix/api/views/event.py

```diff
@@ -744,3 +746,31 @@ def perform_update(self, serializer):
             auth=self.request.auth,
             data={"seats": [serializer.instance.pk]},
         )
+
+    @action(methods=["POST"], detail=False)
+    def bulk_block(self, request, *args, **kwargs):
+        s = SeatBulkBlockInputSerializer(
+            data=request.data,
+            context={"event": request.event, "queryset": self.get_queryset()},
+        )
+        s.is_valid(raise_exception=True)
+
+        seats = s.validated_data["seats"]
+        for seat in seats:
+            seat.blocked = True
+        Seat.objects.bulk_update(seats, ["blocked"], batch_size=1000)
+        return Response({})
+
+    @action(methods=["POST"], detail=False)
+    def bulk_unblock(self, request, *args, **kwargs):
+        s = SeatBulkBlockInputSerializer(
+            data=request.data,
+            context={"event": request.event, "queryset": self.get_queryset()},
+        )
+        s.is_valid(raise_exception=True)
+
+        seats = s.validated_data["seats"]
+        for seat in seats:
+            seat.blocked = False
+        Seat.objects.bulk_update(seats, ["blocked"], batch_size=1000)
+        return Response({})
```

Comment:
Would it make sense to combine both into one method?

```suggestion
    def bulk_change_blocked(self, blocked):
        s = SeatBulkBlockInputSerializer(
            data=self.request.data,
            context={"event": self.request.event, "queryset": self.get_queryset()},
        )
        s.is_valid(raise_exception=True)

        seats = s.validated_data["seats"]
        for seat in seats:
            seat.blocked = blocked
        Seat.objects.bulk_update(seats, ["blocked"], batch_size=1000)
        return Response({})

    @action(methods=["POST"], detail=False)
    def bulk_block(self, request, *args, **kwargs):
        return self.bulk_change_blocked(True)

    @action(methods=["POST"], detail=False)
    def bulk_unblock(self, request, *args, **kwargs):
        return self.bulk_change_blocked(False)
```